app/models/dhs1171_pdf_member_attributes.rb

```diff
@@ -0,0 +1,49 @@
+class Dhs1171PdfMemberAttributes
```

Comment:
I can surmise that Dhs21171 is the form name - haha, that's a wild guess! But I'm wondering what you think about a more generic name that encompasses what the form *is*. It's the snap application for our purposes, so could we name these classes after that instead of the formal, esoteric, Dhs 1171?